platform/engineering/frontend/eslint/eslint-testing.md

```diff
@@ -0,0 +1,57 @@
+# ESLint Rule Changes and Testing
+
+**Author:** Jhonny Gonzalez
+**Last Updated:** February 21, 2020
+**eslint version:** 4.18.2
+
+## Overview
+
+This is an initial process to test and deploy new rules for ESLint.
+
+## Objective
+
+To improve the code coverage and code quality in the repo by enforcing new ESLint rules. This also will help all VFS teams to write better code.
+
+## Background
+
+We installed ESLint a couple of years ago in the VA repo and hasn't been updated nor improved since then.
+
+Now, that VFS teams will have an increased ownership on their pull request reviews, this will help them getting a better judgement as well as facilitate the review process.
```

Comment:
```suggestion
Now that VFS teams will have an increased ownership on their pull request reviews, this will help them get a better judgement as well as facilitate the review process.
```

I'm leaning toward something more like
> The Veteran-facing Service Platform (VSP) has been reviewing all Veteran-facing Services Application (VSA) teams' Pull Requests (PRs) on `vets-website`. Because of the limited nature of the PR reviews VSP engineers can give, one of the main types of feedback we have been able to give concerned code quality.
>
> Because VSP is moving away from reviewing all VSA PRs, we plan to better leverage ESLint to automate that code quality feedback.

What do you think?

Reply:
I love it! Definitely better explained.